Ports/carl/package.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=carl
+version=1.5
+workdir=cryanc-"${version}"
+files="https://github.com/classilla/cryanc/archive/refs/tags/${version}.tar.gz cryanc-${version}.tar.gz f2cae13addf4ed7cb7af3d06069cf082"
+
+build() {
+    run $CC -O3 carl.c -o carl
+}
+
+install() {
+    run cp carl "${SERENITY_BUILD_DIR}/Root/bin"
```

Comment:
Please install ports in `/usr/local/bin`

Reply:
Sounds good. Do you think usr/local should be in Base so ports won't have to make it?